ci/build.sh

```diff
@@ -11,12 +11,8 @@ PROJECT_DIR="$(pwd)"
 markStartOfBlock "$0"
 
 markStartOfBlock "Building Tools"
-"${MSBUILD}" tools/ImpNuget/ImpNuget.csproj //property:Configuration=Release //clp:ErrorsOnly //nologo //m 1>&2
-pushd "tools/DocsLinter"
-    "../../bin/ImpNuget/ImpNuget.exe"
-popd
 
-"${MSBUILD}" tools/DocsLinter/DocsLinter.csproj //property:Configuration=Release //clp:ErrorsOnly //nologo //m 1>&2
+dotnet build tools/DocsLinter/DocsLinter.csproj
```

Comment:
This is missing the `-c Release` flag

Reply:
Since this is technically testing if the projects can be build, maybe `--no-incremental` should be passed to do a full rebuild?